cooperator/models/subscription_request.py

```diff
@@ -108,7 +110,10 @@ def create(self, vals):
         if not cooperator.cooperator:
             cooperator.write({"cooperator": True})
 
-        subscription_request = super().create(vals)
+        # keep only values related to partner
+        partner_vals = {key: value for (key, value) in vals.items() if key in _REQUIRED}
+
+        subscription_request = super().create(partner_vals)
```

Comment:
I think like the idea of this change, but I'm not sure it works well.

First it's not very extendible. If another module wants to write more fields, they would have to append those fields to `odoo.addons.cooperator.models.subscription_request._REQUIRED`, which is not ideal, and would probably break multi-server setups. Normally they could instead override `get_required_field` [sic, typo], but that won't work here because `create` is `@api.model`.

It's also underdocumented. Why is this done? Why do we remove some vals? I suspect there's a good reason. Is it because of some code passing invalid vals? Can that code be fixed downstream?